src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -188,6 +188,16 @@
   private WebElement footer;
   @FindBy(css = "#globalNavigation")
   private WebElement globalNavigationBar;
+  @FindBy(css = "#WikiaPageHeader > nav.wikia-menu-button > span.drop")
+  private WebElement getEditButton;
```

Comment:
the fields are named with nouns like editButton. getEditButton is a verb used for naming methods